src/source_map_utils.ts

```diff
@@ -6,9 +6,31 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {BasicSourceMapConsumer, RawSourceMap, SourceMapConsumer, SourceMapGenerator} from 'source-map';
+import {RawSourceMap, SourceMapConsumer, SourceMapGenerator} from 'source-map';
 import * as ts from './typescript';
 
+/**
+ * This interface was defined in @types/source-map but is absent from the typings
+ * distributed in the source-map package.
+ * Copied from https://unpkg.com/@types/source-map@0.5.2/index.d.ts
+ * see https://github.com/angular/tsickle/issues/750
```

Comment:
leave a TODO with instructions on how to fix eventually?

Reply:
I don't think there's any action to take, probably not worth making a fix upstream for their typings? Maybe they prefer these APIs be undocumented. I'll open an issue at least...

Reply:
Ah, I'm wrong, https://github.com/mozilla/source-map/commit/7c06ac83dd6d75e65f71727184a2d8630a15bf58#diff-7945f6bb445d956794564e098ef20bb3 adds the `toJSON` method among other things.
However that's only in the 0.7 release which adds more breaking changes, eg. the `SourceMapConsumer` constructor is now a sync (???)
So I think we should land this, I'll update the comments.